crates/rattler_cache/src/run_exports_cache/cache_key.rs

```diff
@@ -0,0 +1,87 @@
+use rattler_conda_types::{package::ArchiveIdentifier, PackageRecord};
+use rattler_digest::{Md5Hash, Sha256Hash};
+use std::fmt::{Display, Formatter};
+
+/// Provides a unique identifier for packages in the cache.
+/// TODO: This could not be unique over multiple subdir. How to handle?
```

Comment:
Is this smething that should still be handled?

Reply:
good catch! I think it has already been handled because we are using sha256/md5 to identify them uniquely.